Lib/test/test_pathlib.py

```diff
@@ -938,6 +941,11 @@ def test_match_common(self):
         self.assertTrue(P('B.py').match('b.PY'))
         self.assertTrue(P('c:/a/B.Py').match('C:/A/*.pY'))
         self.assertTrue(P('//Some/Share/B.Py').match('//somE/sharE/*.pY'))
+        # Case-sensitivity
+        self.assertFalse(P('A.py').match('a.PY', case_sensitive=True))
+        self.assertFalse(P('c:/a/B.Py').match('C:/A/*.pY', case_sensitive=True))
+        self.assertFalse(P('c:/a/B/c.PY').match('C:/A/**/*.pY', case_sensitive=True))
+        self.assertTrue(P('c:/a/B/c.pY').match('c:/a/**/*.pY', case_sensitive=True))
```

Comment:
`**` segments aren't properly supported in `match()` at the moment, so I'd rather we don't use them here.